### ISSUE TITLE
need a manifest file to preserve provenance info [important]

### ISSUE THREAD

**Issue by rossmounce** [ContentMine/getpapers#42]
PMR and I strongly think we need a manifest.json of some sort to document in each search.

* the time and date and API used of the getpapers search and the search parameters
* a full listing of all files downloaded from the getpapers query

hence the suggested name of either 'manifest' or 'metadata' for this new JSON file.

If I do a search today, just by looking at the output I will have no idea in 7 days time what search I ran to get those results. PMR also thinks it's very important for downstream tools to have a manifest of all the files in the cmdir.

**Comment by Blahah** [comment]
Yes this is a good idea. It's basically a log that we want, with a bit of structure that can be shared between getpapers, quickscrape, norma and ami. Simple to implement.
